lib/ts/Layout.cc

```diff
@@ -41,56 +47,61 @@ Layout::get()
 }
 
 void
-Layout::create(const char *prefix)
+Layout::create(ts::string_view prefix)
```

Comment:
@jpeach I was talking to @SolidWallOfCode  about this. I am of the view at this point that the "argument" being used to pass std::string_view by value is a micro optimization that does not always work as expected. It make C++ look complex, when it should not. So I am 100% for pass `ts:string_view const& `  or `ts::string_view &` instead as I think this is consistent with how all object  in c++ land should be passed